itest/lnd_misc_test.go

```diff
@@ -230,6 +230,49 @@ func testSphinxReplayPersistence(ht *lntest.HarnessTest) {
 	ht.ForceCloseChannel(carol, chanPoint)
 }
 
+// testFundingExpiryBlocksOnPending checks that after an OpenChannel, and
+// before the funding transaction is confirmed, that the FundingExpiryBlocks
+// field of a PendingChannels decreases.
+func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
+	// Configure bob to require 5 confs for a channel to be considered open
+	alice, bob := ht.Alice, ht.Bob
+	const numConfs = 5
+	args := []string{
+		fmt.Sprintf("--bitcoin.defaultchanconfs=%v", numConfs),
+	}
+	ht.RestartNodeWithExtraArgs(bob, args)
+
+	ht.ConnectNodes(alice, bob)
+	param := lntest.OpenChannelParams{
+		Amt:     btcutil.Amount(100000),
```

Comment:
nit: no need to use `btcutil.Amount` if using a constant. Also a zero value is the default, so no need for the `PushAmt` field either.